### PR TITLE
Handled the Rhythm card when no data is present

### LINKED ISSUE #5468: When the patient has no rhythm the card is fully empty
**Describe the bug**
When the patient has no rhythm the card is fully empty

**To Reproduce**

1. Go to https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/dd001297-f509-4cdf-b1e8-7986ab5c7ced/consultation/1f8c13b9-aabb-4ae6-8662-5c6ff4b906bd/summary
2. Scroll down to Rythm Card
3. See the error

**Expected behavior**
There should be a message that no rhythm information is available.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by nikhilagastya**
- Fixes #5468 
- Added a check before using the map

### CODE REVIEW

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@nikhilagastya 
image
image

make it center aligned

**Comment by nikhilagastya** [comment]
can you review it once @nihal467 !

**Comment by nihal467** [comment]
@nikhilagastya 
image

currently, when there is data, there is a lot of white space coming in both side

@Ashesh3 can you review and post a change request

File: src/Components/Facility/Consultations/PrimaryParametersPlot.tsx [hunk 1]
```diff
@@ -196,10 +196,10 @@ export const PrimaryParametersPlot = ({
             high={60}
           />
         </div>
-        <div className="md:w-full overflow-x-auto pt-4 m-2 px-4 bg-white border rounded-lg shadow h-80">
-          <h3 className="text-sm px-3 mb-2">Rhythm</h3>
+        <div className="md:w-full flex justify-center overflow-x-auto pt-4 m-2 px-4 bg-white border rounded-lg shadow h-80">
```
Comment (Ashesh3) [inline src/Components/Facility/Consultations/PrimaryParametersPlot.tsx]: @nikhilagastya We want the element taking up full width and centered. 

When there is no data available, we want the message to be at the center:

image

When there is data, we want it to take full width, as mentioned by Nihal in the previous message. Try not to have padding/margins on x axis to center the elements, just setting the item's alignment to center should work. Let it take full width.

File: src/Components/Facility/Consultations/PrimaryParametersPlot.tsx [hunk 2]
```diff
@@ -195,66 +196,70 @@ export const PrimaryParametersPlot = ({
...
-                    <div className="relative pb-8">
-                      {rhythmIdx !== obj[1].length ? (
-                        <span
-                          className="absolute top-4 left-4 -ml-px h-full w-0.5 bg-gray-200"
-                          aria-hidden="true"
-                        />
-                      ) : null}
-                      <div className="relative flex space-x-3">
-                        <div>
+        <div className="md:w-full flex justify-center overflow-x-auto pt-4 m-2 px-4 bg-white border rounded-lg shadow h-80">
```
Comment (rithviknishad) [inline src/Components/Facility/Consultations/PrimaryParametersPlot.tsx]:
```suggestion
        <div className="md:w-full flex overflow-x-auto pt-4 m-2 px-4 bg-white border rounded-lg shadow h-80">
```

**Review by Ashesh3 — CHANGES_REQUESTED** [review]

File: src/Components/Facility/Consultations/PrimaryParametersPlot.tsx [hunk 3]
```diff
@@ -195,66 +196,70 @@ export const PrimaryParametersPlot = ({
...
+          <h3 className="text-sm text-center">Rhythm</h3>
+          {Object.keys(rhythmValues).length === 0 ? (
+            <p className="px-4 self-center">No Rhythm data available.</p>
```
Comment (rithviknishad) [inline src/Components/Facility/Consultations/PrimaryParametersPlot.tsx]:
```suggestion
            <p className="mx-auto self-center">No Rhythm data available.</p>
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by nikhilagastya** [comment]
@nihal467 @Ashesh3 @rithviknishad is it okay now ?

**Comment by nihal467** [comment]
@rithviknishad @nikhilagastya 
image

the bug still persist

**Comment by Ashesh3** [comment]
@nihal467 Made the changes.

image
image

image
image
image

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
@Ashesh3 @rithviknishad can you review this pr again

**Review by Ashesh3 — APPROVED** [review]

**Pull request merged**